server/routes/kanban.js

```diff
@@ -9,52 +10,96 @@ function isAuthenticated(req, res, next) {
   }
 }
 
-router.route('/') //make card
-  .post(isAuthenticated, function (req, res) {
+router.route('/')
+  .post(function (req, res) { //make card
+    console.log(req.body);
+
     Card.forge({
       title: req.body.title,
       body: req.body.body,
       priority_id: req.body.priority_id,
       status_id: req.body.status_id,
-      created_by: req.user.id,
+      created_by: req.body.created_by,
       assigned_to: req.body.assigned_to
     }).save()
-      .then(function (err) {
-        res.json({ success: true });
+      .then(function (newCard) {
+        Card.where('id', newCard.id).fetch({
+          columns: ['id', 'title', 'body', 'priority_id', 'status_id', 'created_by', 'assigned_to'],
+          withRelated: [{
+            'assignedUser': function (qb) {
```

Comment:
You should be defining this in the Card class rather than doing it here. If you do it this way, you'll need to specify the function every time you need `assignedUser`. Same for the `createdByUser` below.